engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -80,6 +80,8 @@
   public static final String SER_FILE_NAME = "dt-conf.ser";
   public static final String LAUNCH_CONFIG_FILE_NAME = "dt-launch-config.xml";
   private static final transient AtomicInteger logicalOperatorSequencer = new AtomicInteger();
+  public static final long NODE_BLACKLIST_REMOVAL_TIME = 60 * 60 * 1000;
```

Comment:
What is the unit for BLACKLIST_REMOVAL_TIME, name should include this for clarity.